generators/client/templates/_package.json

```diff
@@ -8,6 +8,7 @@
     "<%= MAIN_SRC_DIR %>bower_components"
   ],
   "devDependencies": {
+    "bower": "1.7.9",
```

Comment:
See my remaks on the issue ... don't think there's much point anymore in a seperate CI profile (would only complicate things more). 
I would keep the plugin execution where it downloads node.
There's a one-time penalty of 10secs for downloading node in the very first mvn -Pprod build.
Subsequent runs will run just as fast as before.